packages/interfaces/src/protocols.ts

```diff
@@ -106,6 +106,13 @@ export type ProtocolCreateOptions = {
    * List of peers to use to bootstrap the node. Ignored if defaultBootstrap is set to true.
    */
   bootstrapPeers?: string[];
+  /**
+   * List of nodes' multiaddrs as strings to use for each protocol. If not specified, random nodes will be used.
+   * This should be used only if you know what you are doing.
+   */
+  nodeToUse?: {
```

Comment:
can you create `store` field with following type:
`store: Partial<StoreProtocolOptions>`

and `StoreProtocolOptions` can have:
```typescript
type StoreProtocolOptions = {
  peers: [],
};
```

example for Filter in one of open PRs - https://github.com/waku-org/js-waku/pull/2193/commits/09127dfc9187bfc15dcd35330553de0de602e093

Reply:
9c98cc47d44d488f70647a1b5fb2693b18b8e9e9